packages/manager/modules/pci/src/projects/new/components/voucher/index.html

```diff
@@ -54,24 +56,26 @@
                 <oui-button
                     data-type="submit"
                     data-aria-label="{{ :: 'pci_projects_new_voucher_form_add_title' | translate }}"
-                    data-disabled="$ctrl.loading.check || !$ctrl.model.voucher.value || $ctrl.globalLoading.finalize"
-                    data-ng-if="!$ctrl.model.voucher.valid"
+                    data-disabled="!$ctrl.canAddVoucher()"
                 >
                     <span
                         data-translate="pci_projects_new_voucher_form_add"
                     ></span>
                 </oui-button>
+            </div>
+            <div class="d-flex flex-row align-items-center justify-content-between mt-1"
+                 data-ng-if="$ctrl.voucherEligibility.voucher">
 
-                <oui-button
-                    data-type="reset"
-                    data-aria-label="{{ :: 'pci_projects_new_voucher_form_remove_title' | translate }}"
-                    data-ng-if="$ctrl.model.voucher.value && $ctrl.model.voucher.valid"
-                    data-ng-click="$ctrl.onVoucherFormReset()"
-                    data-ng-disabled="$ctrl.loading.reset || $ctrl.globalLoading.finalize"
+                <!--Voucher details-->
+                <span data-translate="pci_projects_new_voucher_form_field_add_success"
+                      data-translate-values="{ credit: $ctrl.getFormatCreditText() }"
                 >
-                    <span
-                        data-translate="pci_projects_new_voucher_form_remove"
-                    ></span>
+                </span>
+
+                <!--Reset voucher-->
+                <oui-button size="l" variant="link" disabled="$ctrl.loading.reset"
+                            data-ng-click="$ctrl.onVoucherFormReset()">
```

Comment:
**suggestion:** According to component documentation we should use:
```suggestion
                            on-click="$ctrl.onVoucherFormReset()">
```

Reply:
It was there, but well see :)